pkg/k8s/kubectl.go

```diff
@@ -26,6 +26,11 @@ type kubectl struct {
 	proxyPort int
 }
 
+type VersionInfo struct {
+	Client [3]int
+	Server [3]int
+}
+
 const (
```

Comment:
I don't think there's an actual convention around this, to TIOLI, but I generally prefer constants to be on the top of the file